css/theme/stanford-person.css

```diff
@@ -235,31 +237,47 @@ figcaption {
   .content-type-person_profile a {
     text-decoration: underline; }
   .content-type-person__profilelink a {
-    font-size: 22px; }
+    font-size: 20px; }
   .content-type-person_bio {
     border: 1px solid #dad7cb;
-    padding: 40px;
-    clear: both; }
+    clear: both;
+    margin-bottom: 15px;
+    padding: 40px; }
     .content-type-person_bio p {
       font-size: 18px; }
       @media only screen and (min-width: 768px) {
         .content-type-person_bio p {
           font-size: 22px; } }
+  .content-type-person a {
+    text-decoration: none; }
+  .content-type-person .earth-people__subtitle {
+    margin-bottom: 10px; }
+    .content-type-person .earth-people__subtitle:before {
+      content: "•";
+      margin-right: 20px; }
```

Comment:
Let's talk about this. We might want instead to make it a proper list in twig?

Reply:
@buttonwillowsix,
That makes sense. Wanna show me how to make it happen?

Reply:
I went in to look to see if this could be added to the template, but it is a little more involved than I had hoped. I am making another ticket for this.